app/src/main/res/drawable/ic_key_16dp.xml

```diff
@@ -0,0 +1,20 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
```

Comment:
Can we make height and width to 20 dp?

Reply:
Since it is a vector drawable you can define any height and width directly from the `imageView`. 
Btw, Android stands those kind of icon to 24dp. What are the `heights ` and `widths` that are you defining for that new icons?

Reply:
@filipedelimabrito We can change the width and height in imageview but I'm using those icons in button drawable(In which it would we bit difficult to change width and height).

Reply:
In design the icon size is 20dp, that's why I think It's better to make it 20dp.

Reply:
I'll change it 👍